app/Widgets/Socials_Feed_Widget.php

```diff
@@ -0,0 +1,114 @@
+<?php
```

Comment:
I think this widget should be a part of framework. otherwise - we will have some dummy file in all our generated themes and you will need to remove it in 80% of cases.

same goes to templates. copies should be added to the framework views/widgets/ folder.

inside widget() method you should try to locate template in a theme, if not found - load from framework.